saito-js/lib/custom/shared_methods.web.ts

```diff
@@ -116,11 +116,22 @@ export default class WebSharedMethods extends CustomSharedMethods {
 
   sendMessageToAll(buffer: Uint8Array, exceptions: Array<bigint>): void {
     // console.debug("sending message to  all with size : " + buffer.byteLength);
+    // console.info(' --- Sending to All ---')
     Saito.getInstance().sockets.forEach((socket, key) => {
       if (exceptions.includes(key)) {
         return;
       }
-      socket.send(buffer);
+      try {
+        if (socket.readyState !== socket.OPEN) {
+          socket.onopen = () => socket.send(buffer);
```

Comment:
this won't work either since it's replacing the previous sendMessageToAll's socket.onopen call

Reply:
I have updated the code to just log attempting to send without an open socket as discussed.